src/plugins/dashboard/public/dashboard_top_nav/internal_dashboard_top_nav.tsx

```diff
@@ -118,6 +118,13 @@ export function InternalDashboardTopNav({
     return getDashboardTitle(title, viewMode, !lastSavedId);
   }, [title, viewMode, lastSavedId]);
 
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
... isLoading is only true on the first load, but when refreshing, changing time range or updating query it doesnt switch back to True. Needs to be fixed.